content/pt-br/docs/tasks/configure-pod-container/assign-cpu-resource.md

```diff
@@ -0,0 +1,270 @@
+```
+
+Verifique se o Pod está executando:
+
+```shell
+kubectl get pod cpu-demo --namespace=cpu-example
+```
+
+Ver informação detalhada sobre o Pod:
+
+```shell
+kubectl get pod cpu-demo --output=yaml --namespace=cpu-example
+```
+
+A saída mostra que este contêiner no Pod tem um requisito de CPU de 500 miliCPU e um limite de CPU de 1 CPU.
+
+```yaml
+resources:
+  limits:
+    cpu: "1"
+  requests:
+    cpu: 500m
+```
+Use `kubectl top` para buscar as métricas do pod:
+
+
+```shell
+kubectl top pod cpu-demo --namespace=cpu-example
+```
+
+Este exemplo de saída mostra que o Pod está usando 974 miliCPU, 
+que é ligeiramente menor que o limite de 1 CPU especificado na configuração do Pod.
+
+```
+NAME                        CPU(cores)   MEMORY(bytes)
+cpu-demo                    974m         <something>
+```
+
+Lembre-se que ao especificar `-cpu "2"`, você configurou o Contêiner para tentar 
+usar 2 CPUs, mas o Contêiner está apenas habilitado a usar cerca de 1 CPU. 
+O uso de CPU dos contêineres está sendo limitado, 
+porque o contêiner está tentando usar mais recursos de CPU que o seu limite. 
+
+{{< note >}}
+Outra possível explicação para o uso de CPU estar abaixo de 1,0, é que o Nó 
+não dispõe de recuros de CPU suficientes disponíveis. Lembre-se que o pré-requisito 
+para este exercício, é que o seu cluster tenha disponível ao menos 1 CPU para uso. 
+Se o seu Contêiner executa em um Nó que tem apenas 1 CPU, o Contêiner não pode usar 
+mais que 1 CPU, sem contar o limite de CPU especificado para o Contêiner. 
+
+{{< /note >}}
+
+## Unidades de CPU
+
+Os recursos de CPU são medidos em unidades de *CPU*. Uma CPU, em Kubernetes, é equivalente a:
+
+* 1 AWS vCPU
+* 1 GCP Core
+* 1 Azure vCore
+* 1 Hyperthread em um processador Intel bare-metal com Hyperthreading
+
+
+Valores fracionários são aceitos. Ao Contêiner que requer 0,5 CPU é garantido 
+pelo menos a metade da CPU que um Contêiner que requer 1 CPU. 
+Você pode usar o sufixo m para representar mili. Por exemplo 100m CPU, 100 miliCPU, 
+e 0,1 CPU são todas a mesma. Precisão menor que 1m não é aceita.
+
+CPU é sempre requisitada como uma quantidade absoluta, nunca como uma quantidade relativa; 0,1 é a mesma quantidade de CPU em uma máquina single-core, dual-core, ou 48-core.
+
+Apague seu Pod:
+
+```shell
+kubectl delete pod cpu-demo --namespace=cpu-example
+```
+
+## Especifique um requisito de CPU que é muito grande para seus Nós
+
+Requisitos e limites de CPU são associados aos Contêineres, mas é útil pensar num Pod como tendo requisito e limite de CPU. O requisito de CPU para um Pod é a soma dos requisitos de CPU de todos os Contêineres no Pod. Da mesma forma, o limite de CPU para um Pod é a soma dos limites de CPU para todos os Contêineres no Pod. 
+
+O agendamento do Pod é baseado em requisições. Um Pod é agendado para executar em um Nó, somente se o Nó possuir suficiente recursos de CPU disponíveis para satisfazer os requisitos de CPU do Pod.
```

Comment:
Minor: instead of "suficiente recursos", "recursos de CPU suficientes" would sound better.